server/store/sqlstore/store.go

```diff
@@ -871,69 +879,110 @@ func (s *SQLStore) StoreOAuth2State(state string) error {
 	return nil
 }
 
-func (s *SQLStore) GetStats(remoteID, preferenceCategory string) (*storemodels.Stats, error) {
-	query := s.getQueryBuilder().Select("count(mmChannelID)").From(linksTableName)
-	row := query.QueryRow()
+func (s *SQLStore) GetStats(opts storemodels.GetStatsOptions) (*storemodels.Stats, error) {
```

Comment:
Given these are mostly distinct queries, would it be clearer if we just exposed `GetLinkedChannelsCount`, `GetConnectedUsersCount`, etc.?